libMBIN/Source/NMS/BaseTypes/Colour.cs

```diff
@@ -1,33 +1,104 @@
+    /// </summary>
     [NMS(Alignment = 0x10, Size = 0x10)]
-    [NMSDescription(
-        @"This class is a simple colour one. The values of each field must be between 0 and 1.
-        To convert from the usual representation of a value between 0 and 255, you just need to divide the value
-        by 255 to get the floating point representation used by the game."
-    )]
     public class Colour : NMSTemplate
     {
-        [NMSDescription("The Red component of the colour")]
+        /// <summary>
+        /// The Red component of the colour.
+        /// </summary>
         public float R;
-        [NMSDescription("The Green component of the colour")]
+
+        /// <summary>
+        /// The Green component of the colour.
+        /// </summary>
         public float G;
-        [NMSDescription("The Blue component of the colour")]
+
+        /// <summary>
+        /// The Blue component of the colour.
+        /// </summary>
         public float B;
-        [NMSDescription("The Alpha component of the colour")]
+
+        /// <summary>
+        /// The Alpha component of the colour.
+        /// </summary>
         public float A;
 
-        public Colour(float R, float G, float B, float A)
+
+        /// <summary>
+        /// Creates a Color while providing it with float RGB values.
+        /// <br/>Values must be between 0 - 1.
+        /// </summary>
+        /// <param name="R">Red component of the colour. Value can be anything between 0 and 1.</param>
+        /// <param name="G">Green component of the colour. Value can be anything between 0 and 1.</param>
+        /// <param name="B">Blue component of the colour. Value can be anything between 0 and 1.</param>
+        /// <param name="A">Alpha component of the colour. Value can be anything between 0 and 1.</param>
+        public Colour(float R, float G, float B, float A = 1f)
         {
             this.R = R;
             this.G = G;
             this.B = B;
             this.A = A;
         }
 
+        /// <summary>
+        /// Creates a Color while providing it with standard RGB values.
+        /// <br/>Values must be whole numbers between 0 - 255.
+        /// </summary>
+        /// <param name="R">Red component of the colour. Value can be any whole number between 0 and 255.</param>
+        /// <param name="G">Green component of the colour. Value can be any whole number between 0 and 255.</param>
+        /// <param name="B">Blue component of the colour. Value can be any whole number between 0 and 255.</param>
+        /// <param name="A">Alpha component of the colour. Value can be anything between 0 and 1.</param>
+        public Colour(int R, int G, int B, float A = 1f)
+        {
+            this.R = R / 255;
+            this.G = G / 255;
+            this.B = B / 255;
+            this.A = A;
+        }
+
         public Colour() { }
+
+        /// <summary>
+        /// Returns a formatted string for this Color.
```

Comment:
Might be good to mention that the format will be NMS-style colour, (ie. floats for all values)

Reply:
Done